packages/salesforcedx-utils-vscode/src/helpers/utils.ts

```diff
@@ -59,6 +60,21 @@ export const flushFilePath = (filePath: string): string => {
   return nativePath;
 };
 
+const isSymbolicLink = (path: string) => {
```

Comment:
@mingxuanzhangsfdx, nice job tracking this bug down. I had not considered flushFilepath as the culprit.

I would suggest that you try to find a more efficient means to detect a symlinked directory, other than climbing the path iteratively.